<?xml version="1.0"?>
<launch>

    <arg name="fcu_conn" default="usb"/>
    <arg name="fcu_ip" default="127.0.0.1"/>
    <arg name="gcs_bridge" default="tcp"/>
	<!-- Pixhawk setup -->
    <include file="$(find px4_ground)/launch/1setup.launch">
        <arg name="fcu_conn" default="$(arg fcu_conn)"/>
        <arg name="fcu_ip" default="$(arg fcu_ip)"/>
        <arg name="gcs_bridge" default="$(arg gcs_bridge)"/>
    </include>

    <!-- Set the speed range of the car's waypoint to [0,1] -->
	<node pkg="px4_control" type="mission_car_offboard_node" name="mission_car_offboard_node" output="screen">
		<param name="step1_Pose_x" value = "0"/>	
		<param name="step1_Pose_y" value = "2"/>

		<param name="step2_Pose_x" value = "1"/>	
		<param name="step2_Pose_y" value = "2"/>

		<param name="step3_Pose_x" value = "1"/>	
		<param name="step3_Pose_y" value = "0"/>

		<param name="step4_Pose_x" value = "0"/>	
		<param name="step4_Pose_y" value = "0"/>

		<param name="desire_vel_" value = "0.7"/>
	</node>

</launch>